packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -308,7 +345,7 @@ export default decorate([
         <div>
           <h2>{_('remoteTypeLocal')}</h2>
           <SortedTable
-            collection={remotes.file}
+            collection={state.remoteWithInfo.file}
```

Comment:
Why is `remoteWithInfo` not used with other types?

Reply:
Because the others are not yet fully implemented. So, i change only for file.